api-report/odsp-driver.api.md

```diff
@@ -63,7 +63,7 @@ export function getApiRoot(origin: string): string;
 export function getHashedDocumentId(driveId: string, itemId: string): Promise<string>;
 
 // @public
-export function getLocatorFromOdspUrl(url: URL): OdspFluidDataStoreLocator | undefined;
+export function getLocatorFromOdspUrl(url: URL, requireFluidSignature?: boolean): OdspFluidDataStoreLocator | undefined;
```

Comment:
This documentation does not seem accurate. It leaves an impression that default value is undefined if omitted while in fact it is truthy.